proposals/stage-3/CPC_CHARTER/CPC-CHARTER.md

```diff
@@ -0,0 +1,320 @@
+# Cross Project Council (CPC) Charter
+
+## Section 1. Guiding Principle.
+
+The OpenJS Foundation will operate transparently, openly,
+collaboratively, and ethically. Project proposals, timelines, and status
+must not merely be open, but also easily visible to outsiders.
+
+## Section 2. Evolution of OpenJS Foundation Governance.
+
+Most large, complex open source communities have both a business and a
+technical governance model. OpenJS Foundation’s technical leadership
+is the Cross Project Council (“CPC”). OpenJS Foundation’s business
+leadership is the Board of Directors (the “Board”).
+
+This Cross Project Council Charter reflects a carefully
+constructed, balanced role for the CPC and the Board in the governance of
+OpenJS Foundation. The charter amendment process is for the
+OpenJS Foundation to propose changes using simple majority of the
+full OpenJS Foundation, the proposed changes being subject to review
+and approval by the Board. The Board may additionally make amendments to
+the OpenJS Foundation charter at any time, though the Board will not interfere
+with day-to-day discussions, votes or meetings
+of the CPC.
+
+## Section 3. Board’s Role in Setting OpenJS Foundation’s Strategic Direction. 
+
+The Board will set the overall CPC Policy. The policy will describe the
+overarching scope of the OpenJS Foundation initiative, OpenJS Foundation 
+Foundation’s technical vision and direction and project release
+expectations in the form of expected cadence and intent. The Board will
+use the CPC as a delegate body for governing technical implementation,
+individual project scope and direction while they remain within the scope
+and direction of the policies as described in the CPC Policy document and
+approved by the Board.
+
+The Board sets the overall CPC policy through [Foundation mission and vision
+statements][] as well as within the [Foundation bylaws][]. The policy describes
+the overarching scope of the OpenJS Foundation initiatives , technical vision,
+and direction. The Board uses the CPC as a delegate body for governing
+high-level technical policy and procedures while remaining within the scope and
+direction of the policies set by the Board.
+
+
+## Section 4. Establishment of the CPC.
+
+The CPC is an inclusive group with three types of participants:
+
+* Observers
+* Regular members
+* Voting members
+
+
+### Observers
+
+Observers are free to attend meetings and participate in the work of the
+CPC as well as the consensus seeking process.
```

Comment:
This grants Observers the ability to block things that have consensus among CPC members. If that's intentional, cool. But if not, uh, you'll want to change that.

Reply:
That is intentional.